src/templates/sysconfig.html

```
@@ -189,23 +189,32 @@ <h2 class='mb-3 p-2 pl-3 section-title'>{{ 'Registration and authentication conf
+    {% set inputId = 'branding-file-' ~ brandingId %}
+
     <h3>
       <span class='text-nowrap'>
-        <button
-          data-action='edit-logo'
-          data-target='{{ key }}'
-          type='button'
-          title='{{ 'Edit'|trans }}'
-          class='btn btn-secondary'
-          aria-label='{{ 'Edit'|trans }}'
+        <label
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)